test/github-issues/7662/issue-7662.ts

```diff
@@ -0,0 +1,105 @@
+import { expect } from "chai";
+import { Connection } from "../../../src/connection/Connection";
+import {
+    createTestingConnections,
+    closeTestingConnections,
+} from "../../utils/test-utils";
+import { MemoryLogger } from "./memory-logger";
+
+describe("github issues > #7662 postgres extensions installation should be optional", () => {
+    it("should NOT install extensions if option is disabled", async () => {
+        let connection: Connection | null = null;
+        try {
+            const connections = await createTestingConnections({
+                entities: [`${__dirname}/entity/*{.js,.ts}`],
+                enabledDrivers: ["postgres"],
+                schemaCreate: false,
+                dropSchema: true,
+                createLogger: () => new MemoryLogger(true),
+                driverSpecific: {
+                    installExtensions: false,
+                },
+            });
+
+            expect(connections).to.have.length(1);
+
+            connection = connections[0];
+
+            const logger = connection.logger as MemoryLogger;
+            const createExtensionQueries = logger.queries.filter((q) =>
+                q.startsWith("CREATE EXTENSION IF NOT EXISTS")
+            );
+
+            expect(createExtensionQueries).to.be.empty;
```

Comment:
This doesn't make sense 🤷 I'm so confused why it's erroring out like that. I can't replicate it locally

<img width="802" alt="Screen Shot 2021-06-24 at 9 33 00 PM" src="https://user-images.githubusercontent.com/2498157/123272228-258dd580-d534-11eb-9771-fea924f254ee.png">

Reply:
If you think it's something odd that only shows up in CI You can always push it with a temporary change to print out the values before the expect?

Reply:
This failed expectation is a few lines above, not this line.

Reply:
AHHH 💡 thanks bro. That was exactly it